API.yaml

```diff
@@ -1664,9 +1663,11 @@ components:
       type: object
       properties:
         index:
+          description: The rank of this line in the virtual table
           type: integer
           format: int64
         cells:
```

Comment:
I like the cells. Maybe we should align the XY/Time Graph to this as well and use "cells" instead of using "labels". In a separate patch of course if we decide for it.